chaosaws/elbv2/actions.py

```diff
@@ -126,6 +126,27 @@ def set_subnets(load_balancer_names: List[str],
     return results
 
 
+def delete(load_balancer_names: List[str],
+           configuration: Configuration = None,
+           secrets: Secrets = None):
+    """
+    Deletes the provided load balancer(s).
+
+    Parameters:
+        - load_balancer_names: a list of load balancer names
+    """
+    client = aws_client('elbv2', configuration, secrets)
+    load_balancers = get_load_balancer_arns(load_balancer_names, client)
+
+    for k, v in load_balancers.items():
+        if k not in ('application', 'network'):
+            continue
+
+        for l in v:
+            logger.debug('Deleting load balancer %s' % l)
+            client.delete_load_balancer(LoadBalancerArn=l)
```

Comment:
Does this return anything we should at least log? Could be handy when things go wrong...

Reply:
from the doc, there is no return on a successful delete, just the `ResponseMetadata`

Reply:
I see. Well, it is what it is :)

Thanks.